common/src/main/java/com/wynntils/features/user/overlays/map/MiniMapFeature.java

```diff
@@ -108,9 +108,11 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
             // enable mask
             switch (maskType) {
                 case Rectangular -> RenderUtils.enableScissor((int) renderX, (int) renderY, (int) width, (int) height);
-                    // case Circle -> {
-                    // TODO
-                    // }
+                case Circle -> {
+                    RenderUtils.createMask(
```

Comment:
Codana is right. We don't need the `{ ... }`. Removing them will also fit with the style of the other switch branch.

Reply:
Best for a general Qodana/formatting pr